build/targets/KoreBuild.DefaultBuildSettings.props

```diff
@@ -1,4 +1,16 @@
 <Project>
+  <!-- pickup build number from CI if available -->
+  <PropertyGroup Condition=" '$(BuildNumber)' == '' ">
+    <!--
+      TeamCity=https://confluence.jetbrains.com/display/TCD10/Predefined+Build+Parameters
+      AppVeyor=https://www.appveyor.com/docs/environment-variables/
+      Travis=https://docs.travis-ci.com/user/environment-variables/
+    -->
+    <BuildNumber Condition=" '$(BUILD_NUMBER)' != '' ">$(BUILD_NUMBER)</BuildNumber>
```

Comment:
BuildCounter? That's not an MSBuild property we use anywhere. Did you mean BuildNumber?

On UniverseCoherence, I manually set env.BuildNumber = %build.number%, but that's the only place I remember adding it. Most CI configs end up producing packages with temporary build numbers because the BuildNumber is unset.